### ISSUE TITLE
CodeDecodeError:'charmap' codec can't decode

### ISSUE THREAD

**Issue by nidaaz** [ArtemKushnerov/az#36]
Hi!
There's an error when I try to download the app. I use the input file from Androzoo and the file format already CSV.

![image](https://user-images.githubusercontent.com/57988733/98891139-235c9680-24d0-11eb-881c-341a815a4820.png)

This is my code when I try to download

![image](https://user-images.githubusercontent.com/57988733/98891308-851d0080-24d0-11eb-8a63-b5c52e6e6b20.png)

What should I do to fix it?

**Comment by tleadon** [comment]
I'm getting the same error. :( 
![image](https://user-images.githubusercontent.com/75650693/151060312-6b4bb46f-0c74-4d49-bdd5-68f330509757.png)

**Comment by kallix** [comment]
Did you decompress the latest.csv.gz file from AndroZoo?
AZ is unable to directly process the gzipped file provided by AndroZoo. It must be decompressed manually.

**Comment by tleadon** [comment]
Yup! I decompressed it into a regular csv and used that as the input file the .az points to.

**Comment by kallix** [comment]
Have you tried different versions of the file at https://androzoo.uni.lu/static/lists/latest.csv.gz ? (it is recomputed every night)

**Comment by tleadon** [comment]
I'll try it! Thank you for the suggestion.

**Comment by EngAshDev** [comment]
hi im getting the same error , but i tried diffrent version of Latest.csv
one gave me this error {{  return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 6662: character maps to <undefined> }}
the othere one gave me {{ return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 445: character maps to <undefined> }}

**Comment by EngAshDev** [comment]
im using conda env / python 3.6 / az 1.3.1
i also tried az 1.3.0 / 1.2.2 / python 3.6
but still no luck!
